docs/docusaurus/docs/core/connect_to_data/filesystem_data/_create_a_batch_definition/_examples/_directory_partitioned_daily.py

```diff
@@ -20,6 +20,6 @@
 batch = batch_definition.get_batch(
     batch_parameters={"year": "2019", "month": "01", "day": "01"}
 )
-batch.head()
+print(batch.head())
```

Comment:
Note for reviewer: these changes to `batch.head()` aren't showing where I'd expect them to, for example on https://deploy-preview-10560.docs.greatexpectations.io/docs/core/connect_to_data/filesystem_data/ 

Am I missing some step to make the change take effect or am I looking for it in the wrong place?

Reply:
What *are* you seeing?

Reply:
fwiw I don't typically use `head()`, but I just tried, and what I'm seeing is that looking at the value (e.g. in a debugger, you'll see reasonable data, but printing is doing nothing. Seems like a bug to me that print does nothing - may worth filing a ticket on Timber's board. I'd probably leave off the `print` for now.

Reply:
> What are you seeing?

I am seeing `batch.head()` still in the deploy preview code samples. I'm not seeing the change I'm trying to apply to wrap that in `print()`.

It looks like there are at least 4 layers of files involved in the creation of the page where I'm expecting to see changes so I may just be getting my wires crossed but here's a screenshot of one case where I'm not seeing  the expected change
<img width="2346" alt="Screenshot 2024-10-25 at 2 14 18 PM" src="https://github.com/user-attachments/assets/2205e73a-9d7a-4c51-866c-7fdf87f5811e">

From left to right in vs code I have
- the main markdown file for the page at https://deploy-preview-10560.docs.greatexpectations.io/docs/core/connect_to_data/filesystem_data/
- a markdown snippet for the "Create a Batch Definition" section of the page
- a markdown snippet for the "Directory data asset" tab of that section
- a .py file for the code samples that display in the Partioned > Daily subtab of the section. 

Then at the bottom I have what I'm seeing in the deploy preview

---

Am I looking in the wrong place in the deploy preview to see this change take effect? Do I need to do something additional to make the change take effect?

Reply:
after talking to @rachhouse I cleared the netlify cache and retried the build and that fixed things. I now see this. change where I expect to see it